pkg/virt-handler/vm.go

```diff
@@ -1572,43 +1576,109 @@ func (d *VirtualMachineController) heartBeat(interval time.Duration, stopCh chan
 			// Label the node if cpu manager is running on it
 			// This is a temporary workaround until k8s bug #66525 is resolved
 			if d.clusterConfig.CPUManagerEnabled() {
-				d.updateNodeCpuManagerLabel()
+				updateNodeCpuManagerLabel()
 			}
 		}, interval, 1.2, true, stopCh)
 	}
 }
 
-func (d *VirtualMachineController) updateNodeCpuManagerLabel() {
+func (d *VirtualMachineController) getCPUManagerLabelFunc() func() {
+	var lastConfigID string
```

Comment:
hm where do you set value for this variable?
IMHO you should get it from annotation under `dicoverCPUPolicyMachineConf` method and do not send it as a parameter

Reply:
I do that to keep the last id between function calls, to avoid unnecessary API calls, if lastId == currentId (from annotation) then don't do anything more.